### ISSUE TITLE
Name clash

### ISSUE THREAD

**Issue by kkaris** [indralab/mira#42]
There is a name clash between `mira.metamodel.templates.Config` and `mira.dkg.models.Config`

**Comment by bgyori** [comment]
Does this create issues in the documentation?

**Comment by kkaris** [comment]
No, I was just thinking it could tag along with that PR. I should have made that more clear.
